### DISCUSSION TITLE
Neoforge support?

### DISCUSSION THREAD

**Discussion by lukachukai1** [PrismLauncher/PrismLauncher#1509] [Ideas]
When, if ever, will prism launcher begin to support neoforge as well as forge? As forge is going to remain not updated, this seems important.

**Comment by Scrumplex** [comment]
There is a feature request here: https://github.com/PrismLauncher/PrismLauncher/issues/1356

And a pending pull request that implements this feature here: https://github.com/PrismLauncher/PrismLauncher/pull/1498

It is scheduled to be available in Prism Launcher 8.0
